daemon/pod.go

```diff
@@ -42,11 +43,201 @@ type Pod struct {
 	sync.RWMutex
 }
 
-func NewPod(rawSpec []byte, id string, data interface{}) (*Pod, error) {
+// TODO: remove convertToRunvPodSpec after pod.UserPod is deleted from runv
+func convertToRunvPodSpec(podSpec *apitypes.UserPod) (*pod.UserPod, error) {
```

Comment:
apitypes.UserPod and runv.UserPod(shoud be hyperd.UserPod) are both user faced API structure. One for grpc/protobuf, One for json. they will be converted to runv.PodInfo(doesn't exist yet), they are two types of structure services for two types of API. 

Since the runv.UserPod represents the pod spec right now(in json), If user uses grpc api and pod spec to create pod,  the extra conversion from json to protobuf is needed.